app/assets/data/bebbo/sponsers.ts

```diff
@@ -0,0 +1,361 @@
+    },
+    country_national_partner: {
+      name: "MoH Montenegro Logo.png",
+      alt: "Montenegro Partner Logo"
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Montenegro-Montenegrin"
+    ]
+  },
+  {
+    id: 36,
+    type: "Country",
+    title: "North Macedonia",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      localPath: nmSponserLogo,
+      name: "2.png",
+      alt: "Funded by the European Union"
+    },
+    country_national_partner: {
+      name: "1.png",
+      alt: "J3Y WHCTUTYT 3A JABHO 3APABJE HA PEMY5JVKA CEBEPHA MAKE/OHUJA"
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "North Macedonia-Albanian",
+      "North Macedonia-Macedonian"
+    ]
+  },
+  {
+    id: 141,
+    type: "Country",
+    title: "Romania",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Romania-Romanian"
+    ]
+  },
+  {
+    id: 41,
+    type: "Country",
+    title: "Serbia",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      name: "nordeus fondacija logo 1000 wide no spacing final.png",
+      alt: "Nordeus"
+    },
+    country_national_partner: {
+      name: "GZZJZ national partner 1000 wide with spacing.png",
+      alt: "GZZiZ"
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Serbia-Serbian",
+      "Serbia–English"
+    ]
+  },
+  {
+    id: 151,
+    type: "Country",
+    title: "Slovakia",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      name: "PEDIATRI-logarium01av.png",
+      alt: "Pediatri Deťom"
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Slovakia-Slovak"
+    ]
+  },
+  {
+    id: 46,
+    type: "Country",
+    title: "Tajikistan",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      name: "MOH_Tajikistan_Logo_TAJ (2).png",
+      alt: "MoH Tajikistan "
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Tajikistan-Russian",
+      "Tajikistan-Tajik"
+    ]
+  },
+  {
+    id: 136,
+    type: "Country",
+    title: "Ukraine",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Ukraine - Ukrainian"

```

Comment:
Some entries have spaces around the hyphen in the language strings ("Ukraine - Ukrainian"), while others do not ("Ukraine-Ukrainian"). Should we make them consistent throughout the array?